wm/desk.go

```diff
@@ -328,26 +376,38 @@ func (x *x11WM) configureWindow(win xproto.Window, ev xproto.ConfigureRequestEve
 	}
 
 	name := windowName(x.x, win)
-	isRoot := x.root != nil && name == x.root.Title()
-	if isRoot {
-		width = x.x.Screen().WidthInPixels
-		height = x.x.Screen().HeightInPixels
+	for _, screen := range desktop.Instance().Screens().Screens() {
+		if !x.isRootTitle(name) {
+			continue
+		}
+		if screenNameFromRootTitle(name) == screen.Name {
```

Comment:
could this if be inverted and combined with the one above so the main block below goes one level up?